### PR TITLE
Add Vuetify

### PR DESCRIPTION
**Pull request by johnleider**
Thank you!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const authors = {
   hueniverse: 'https://www.patreon.com/eranhammer',
   hughsk: 'https://hughsk.io/donate/',
   jayphelps: 'https://www.patreon.com/jayphelps',
+  johnleider: 'https://www.patreon.com/vuetify',
```
Comment (feross) [inline index.js]: This npm user has no packages: https://www.npmjs.com/~johnleider

It looks like you typo'd your username: https://www.npmjs.com/~johnjleider

File: index.js [hunk 2]
```diff
@@ -282,6 +283,7 @@ const packages = {
   'vue-admin': 'https://opencollective.com/vue-admin',
   'vue-js-modal': 'https://opencollective.com/vue-js-modal',
   'Vulcan': 'https://opencollective.com/telescope',
+  'vuetify': 'https://www.patreon.com/vuetify',
```
Comment (feross) [inline index.js]: Remove this line, there's no need to mention packages explicitly except for team projects. By adding yourself to the `authors` section, your donate link will show up for all the packages you maintain.

File: index.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ const authors = {
   hueniverse: 'https://www.patreon.com/eranhammer',
   hughsk: 'https://hughsk.io/donate/',
   jayphelps: 'https://www.patreon.com/jayphelps',
+  ~johnjleider: 'https://www.patreon.com/vuetify',
```
Comment (feross) [inline index.js]: This isn't valid JS.

**Comment by johnleider** [comment]
Sorry, long day.

File: index.js [hunk 4]
```diff
@@ -16,6 +16,7 @@ const authors = {
   hueniverse: 'https://www.patreon.com/eranhammer',
   hughsk: 'https://hughsk.io/donate/',
   jayphelps: 'https://www.patreon.com/jayphelps',
+  '~johnjleider': 'https://www.patreon.com/vuetify',
```
Comment (feross) [inline index.js]: Lol, sorry this is still wrong. The username doesn't have a `~` in it. You can see how all the other usernames are done right next to yours. I'll just fix this and merge.

**Pull request merged**